### DISCUSSION TITLE
반사를 구현하려 하는데 Reflection Probe 사용이 가능한가요 ?

### DISCUSSION THREAD

**Discussion by xowjd913** [naverz/zepeto-studio-kor#782] [5. Q&A]
제페토 내에서 Reflection Probe 혹시 사용이 가능한가요 ?
사용해보신분 계시면 어떤식으로 사용하셨을까요 ?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 제페토 월드에서 Reflection Probe 사용이 가능합니다.😊
일반적인 유니티에서 Reflection Probe를 적용하는 방법과 동일합니다.
1. 오브젝트에 Material 적용
2. Hierarchy에서 Reflection Probe 추가 
3. 맵 오브젝트를 Reflection Probe Static 로 세팅하기
4. Window > Rendering > Lighting > Bake Reflection Probe 실행

![스크린샷 2022-05-23 오후 4 12 27](https://user-images.githubusercontent.com/100118650/169763734-51366645-7460-4acc-ae9c-af238b354076.png)

![스크린샷 2022-05-23 오후 4 08 12](https://user-images.githubusercontent.com/100118650/169763040-efb3f0b4-e02e-4eba-ab6b-4e818eee31b8.png)
